docs/usage/automation.md

```diff
@@ -201,6 +206,59 @@ When the primary entity you listen to is an incident & then use the "Container W
 #### Share with organizations
 
 Will share every entity in the received STIX 2.1 bundle with Organizations you configured. Your platform needs to have declared a platform main organization in Settings/Parameters.
+You can decice to share only the main triggering element, or the whole bundle thanks to the toogle.
+
+Compared to other components, this component **makes direct call to the database**: this means that the query will be applied before the "send to ingestion" step. As a result, if, **wthin the same playbook**, you attempt to create a new entity (via the wrap in container step) and share the entity, the playbook will fail. Indeed, the entity will not yet be created, since it won't be sent to ingestion yet. You need to share the entity in another playbook to achieve this use case.
+
+#### Unshare with organizations
+
+Will unshare every entity in the received STIX 2.1 bundle with Organizations you configured. Your platform needs to have declared a platform main organization in Settings/Parameters.
+You can decice to unshare only the main triggering element, or the whole bundle thanks to the toogle.
+
+Compared to other components, this component **makes direct call to the database**: this means that the query will be applied before the "send to ingestion" step. As a result, if, **wthin the same playbook**, you attempt to create a new entity (via the wrap in container step) and unshare the entity, the playbook will fail. Indeed, the entity will not yet be created, since it won't be sent to ingestion yet. You need to unshare the entity in another playbook to achieve this use case.
+
+#### Manage Access Restriction
+
+Will apply authorize members on the bundle within the playbook. It is only compatible with entities supporting authorize members (Containers, Drafts).
```

Comment:
isn't it compatible also with Organization?
maybe add a link to [Auth members documenation](https://docs.opencti.io/latest/administration/authorized-members/?h=me) OR detail wich containers may be restricetd ?